internal/action/actions.go

```diff
@@ -1794,24 +1794,24 @@ func (h *BufPane) HalfPageDown() bool {
 
 // ToggleDiffGutter turns the diff gutter off and on
 func (h *BufPane) ToggleDiffGutter() bool {
-	if !h.Buf.Settings["diffgutter"].(bool) {
-		h.Buf.Settings["diffgutter"] = true
+	diffgutter := !h.Buf.Settings["diffgutter"].(bool)
+	h.Buf.SetOptionNative("diffgutter", diffgutter)
+	if diffgutter {
```

Comment:
Again, it would be good if the commit message explained _why_ we need to use `SetOptionNative()`. Again, not because I don't know why but because that's what commit messages are for.